File: app/src/main/java/club/siwoo/doitflow/ui/home/HomeViewModel.java
package club.siwoo.doitflow.ui.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HomeViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public HomeViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("You don't have any thing set up yet! Get Started By Clicking the + Button Down There!");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/club/siwoo/doitflow/ui/home/HomeViewModel.kt
package club.siwoo.doitflow.ui.home

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class HomeViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "Nothing Here! Start Planning by clicking the + Button Below!"
    }
    val text: LiveData<String> = _text
}